src/Form/Type/SyliusGatewayConfigurationType.php

```diff
@@ -4,18 +4,29 @@
 
 namespace Ahmedkhd\SyliusPaymobPlugin\Form\Type;
 
+use BitBag\SyliusCmsPlugin\Entity\MediaInterface;
+use BitBag\SyliusCmsPlugin\Form\Type\MediaAutocompleteChoiceType;
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\File;
+use Symfony\Component\Validator\Constraints\NotNull;
 
 final class SyliusGatewayConfigurationType extends AbstractType
```

Comment:
@ahmed-omani Can you please rename this class to be PaymobGatewayConfigurationType